assets/elasticsearch/reset.py

```
@@ -0,0 +1,5 @@
+import requests
+
+collection_name = "elasticsearch_clp_bench"
+
+requests.delete(f"http://localhost:9202/{collection_name}")
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)